interface-elements-for-desktop/articles/rich-text-editor/track-changes/enable-track-changes.md

```diff
@@ -0,0 +1,59 @@
+# Enable Track Changes
+
+The Rich Text Editor allows you to use Track Changes to make revisions in a document.
+
+## Turn On Track Changes
+
+On the **Review** [tab](text-editor-ui/ribbon-interface.md), in the **Tracking** group, click the **Track Changes** button.
+
+![track-changes-enable](../../../images/rich-text-editor-track-changes-enable.png)
+
+When Track Changes is turned on, changes are indicated by red lines in the margin.
+
+![IMAGE](../../../images/rich-text-editor-track-changes-simple-markup.png)
+
+When it's turned off, the Rich Text Editor stops marking changes, but the lines are still displayed in the document.
```

Comment:
```suggestion
When you turn of Track Changes, the Rich Text Editor does not mark changes but displays the lines in the document.
```

Reply:
Which 'lines' do you refer to?

Reply:
The lines from the previous sentence. 
> When Track Changes is turned on, changes are indicated by red lines in the margin.
The image shows them too.

Reply:
Ok